graph-builder/src/registry.rs

```diff
@@ -280,13 +280,13 @@ pub fn fetch_releases(
 /// Update Images with release metadata should be immutable, but
 /// tags on registry can be mutated at any time. Thus, the cache
 /// is keyed on the hash of tag layers.
-fn cache_release(
+fn cache_release<S: ::std::hash::BuildHasher>(
```

Comment:
This lint mostly makes sense when writing libraries with public API, where the consumer may need to pick the hasher. It isn't super-relevant in our case, I'd just allow it.

Reply:
I understand. However, [the caller of this function](https://github.com/openshift/cincinnati/pull/106/files#diff-991c40183d69ebfda48cb830e94be824R194), which is part of a public API, passes down a variable which has this signature.

Reply:
I'm fine keeping the new form if you like it (it's good practice anyway).

Reply:
My point was that we have to decide for the caller and the callee as they have to match. So if we want this form in the public API we have to use it internally too.